src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/fine/activity/InvokeSyncDebtPositionActivity.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.fine.activity;
```

Comment:
probably this activity will be used by all custom WF
```suggestion
package it.gov.pagopa.pu.workflow.wf.debtposition.custom.activity;
```